src/de/jost_net/JVerein/gui/view/BuchungView.java

```diff
@@ -44,12 +47,55 @@ public void bind() throws Exception
         DokumentationUtil.BUCHUNGEN, false, "question-circle.png");
     if (control.getBuchung().getSpeicherung())
     {
-      buttons.addButton("Neu", new BuchungNeuAction(), null, false, "document-new.png");
+      buttons.addButton("Neu", new BuchungNeuAction(), null, false,
+          "document-new.png");
     }
-    Button savButton = new Button("Speichern",
-        control.getBuchungSpeichernAction(), null, true, "document-save.png");
-    savButton.setEnabled(!buchungabgeschlossen);
-    buttons.addButton(savButton);
+    Button saveButton = null;
+    if (control.getBuchung().getSplitTyp() != null)
+    {
+      Button savebackButton = new Button("Speichern und zur�ck", new Action()
+      {
+        @Override
+        public void handleAction(Object context)
+        {
+          try
+          {
+            control.getBuchungSpeichernAction().handleAction(context);
+            GUI.startView(SplitBuchungView.class.getName(), SplitbuchungsContainer.getMaster());
+          }
+          catch (Exception e)
+          {
+            GUI.getStatusBar().setErrorText(e.getMessage());
+          }
+        }
+      }, null, true, "go-previous.png");
+      savebackButton.setEnabled(!buchungabgeschlossen);
+      buttons.addButton(savebackButton);
+      
+      saveButton = new Button("Speichern und n�chste", new Action()
+      {
+        @Override
+        public void handleAction(Object context)
+        {
+          try
+          {
+            control.getBuchungSpeichernAction().handleAction(context);
+            new SplitbuchungNeuAction().handleAction(context);
+          }
+          catch (Exception e)
+          {
+            GUI.getStatusBar().setErrorText(e.getMessage());
+          }
+        }
+      }, null, true, "document-save.png");
+    }
```

Comment:
Du benutzt oben das go-previous icon, warum hier nicht das go-next icon?

Reply:
Ich finde hier das go.next icon besser weil man wirklich zu einer neuen Buchung kommt. 
Bei zurück hätte ich eher das document-save icon erwartet. Man kommt ja nicht zur zuletzt bearbeiteten Buchung zurück sondern beendet die Buchungsserie. Da verwirrt mich auch etwas das zurück, weil man nicht in der Buchungsserie zurückgeht sondern sie beendet. Aber du meinst zurück zu Splittbuchung. Aber mir fällt auch kein besserer Name ein.

Reply:
Das Problem ist die unterschiedliche Interpretation des zurück weil es auch ein vor gibt. Das interpretiere ich auf den ersten Blick auf gleicher Ebene, also vor und zurück zwischen Buchungen.
Wie wäre es wenn man beim ersten Button das document-save icon nimmt und es nur "Speichern" nennt. Beim rechten nimmt man des go-next Icon und nennt es nur "Weitere Buchung".
Dann steht da kein vor und zurück und es sollte kein Fehlinterpretation geben. Beim ersten Button ist man halt fertig und beim andern macht man eine weitere Buchung.

Reply:
Ja, das ist ein guter Vorschlag, setze ich so um